docs/src/test-sharding-js.md

```diff
@@ -55,7 +55,7 @@ on:
   pull_request:
     branches: [ main, master ]
 jobs:
-  test:
+  playwright-tests:
```

Comment:
can we change the references instead and keep test here?

Reason: Have it as much aligned as [this](https://github.com/microsoft/create-playwright/blob/main/assets/github-actions.yml) as possible, since thats what the users most likely have as a foundation.

Reply:
That was renamed to `playwright-tests` to disambiguate references.